service/tasks/driver.py

```diff
@@ -847,6 +867,9 @@ def destroy_instance(instance_alias, user, core_identity_uuid):
       max_retries=10)
 def deploy_script(driverCls, provider, identity, instance_id,
                   script, **celery_task_args):
+    """
+    #TODO: Should this be ansible now?
+    """
```

Comment:
This is a question I _had_ too. I'm not sure about the answer. But it seems like a reasonable division of labor.